### PR TITLE
718: Put confirmation modal into own component

### PR DESCRIPTION
**Pull request by trbmcginnis**
Modal for prompting user to review and then confirm or cancel their hearing & rec inputs

### CODE REVIEW

**Comment by godfreyyeung** [comment]
Is it possible to move the `modalOpen` variable inside the `confirmation-modal` component? 
The modal component can then keep  track of `modalOpen` by itself.

To let outside controllers/templates interact with the modal, the modal could yield some actions for toggling `modalOpen`

@allthesignals thoughts on component design?

File: app/templates/components/confirmation-modal.hbs [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<div id="hearing-confirmation" class="reveal-overlay" style="display:block;">
```
Comment (andycochran) [inline app/templates/components/confirmation-modal.hbs]: The `id` should be more generic if this is going to be used in places other then the hearing form.

Comment (andycochran) [reply app/templates/components/confirmation-modal.hbs]: Actually, IDK if an `id` is even necessary.

File: app/templates/my-projects/project/recommendations/add.hbs [hunk 2]
```diff
@@ -464,107 +464,105 @@
...
                   {{/if}}
-                  {{#if this.allActions}}
-                    <h5>
-                      Recommendation:
+                {{else}}
+                  {{#each this.dispositionAndChangesetPairs as |dispositionChangesetPair idx|}}
+                    <h5 class="large-margin-top">
+                      Recommendation: {{dispositionChangesetPair.disposition.action.name}}
+                      <small class="dark-gray">
+                        ({{dispositionChangesetPair.disposition.action.ulurpnumber}})
```
Comment (allthesignals) [inline app/templates/my-projects/project/recommendations/add.hbs]: ulurpnumber <- needs to be dcp

**Pull request merged**